src/display/canvas.js

```diff
@@ -705,8 +705,9 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
 
   CanvasGraphics.prototype = {
 
-    beginDrawing: function CanvasGraphics_beginDrawing(transform, viewport,
-                                                       transparency) {
+    beginDrawing({
+      transform, viewport, transparency, background = null,
+    }) {
```

Comment:
This looks kind of weird, compared to existing code (e.g. the placement of the `{` bracket is strange); I think it ought to be:
```javascript
beginDrawing({ transform, viewport, transparency,
               background = null, }) {
```